app/styles/player.styl

```diff
@@ -226,20 +226,35 @@
             display: none
 
         #volume-bar
+            position: absolute
             width: 90px
+            height: 100%
+            border-radius: 2px
+            display: inline-block
+            cursor: pointer
+
+        #volume-bar::before,
+        #volume
+            position: absolute
+            content: ""
+            width: 100%
             height: 4px
             border-radius: 2px
+            top: 34px
```

Comment:
For vertical centering in this context, prefer the use of `transform` rule, so use a

```css
top: 50%
transform: translateY(-50%)
```